Application/EdFi.Ods.Api.IntegrationTestHarness/appsettings.json

```diff
@@ -60,9 +60,25 @@
       {
         "Name": "XsdMetadata",
         "IsEnabled": true
+      },
+      {
+        "Name": "ExternalCache",
+        "IsEnabled": false
       }
     ],
     "ExcludedExtensions": [],
+    "ExternalCache":{
```

Comment:
There is already a section for "Caching". Could this be merged with that?

Reply:
I would agree @vimayya . Possibly putting it inside a Configuration section inside Caching?

Reply:
@vimayya @bptillman I could be combined, but I am concerned it might become confusing that we have cache configuration and an option feature of external cache.  Merging the two seems confusing, but it can be done

Reply:
@vimayya @bptillman I moved the Cache feature into ApiSettings and combined it with external cache.  I also made the provider specific values in provider specific objects for clarity.  Example Redis provider config in a redis object